test.js

```diff
@@ -56,6 +61,78 @@ test('get', t => {
 	F4Class.prototype.foo = 1;
 	const f4 = new F4Class();
 	t.is(dotProp.get(f4, 'foo'), 1); // #46
+
+	t.true(dotProp.get({'': {'': true}}, '..'));
+	t.true(dotProp.get({'': true}, '.'));
```

Comment:
Not sure what these should actually do

Reply:
They should just return `undefined`. No point in supporting weird cases like that.